compiler/src/compiler/cst.rs

```diff
@@ -206,9 +206,9 @@ impl Display for Cst {
                 Ok(())
             }
             CstKind::Text {
-                opening_quote,
+                opening: opening_quote,
                 parts,
-                closing_quote,
+                closing: closing_quote,
             } => {
```

Comment:
Oh, the joy of Rust's automatic refactoring. This should probably be this:

```
CstKind::Text {
  opening,
  parts,
  closing,
} => { ...
```

The same occurs in several other places (also in other files) as well.

Reply:
Resolved by 4854d46